geoengine_partner/__manifest__.py

```diff
@@ -1,37 +1,21 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Author: Nicolas Bessi
-#    Copyright 2011-2012 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2011-2012 Nicolas Bessi <nicolas.bessi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
 {'name': 'Geospatial support of partners',
- 'version': '8.0.0.1.0',
+ 'version': '10.0.1.0.0',
  'category': 'GeoBI',
- 'author': "Camptocamp,Odoo Community Association (OCA)",
+ 'author': "Camptocamp, Odoo Community Association (OCA)",
  'license': 'AGPL-3',
  'website': 'http://openerp.camptocamp.com',
  'depends': [
      'base',
      'base_geoengine'
  ],
  'data': [
-     'geo_partner_view.xml'
+     'views/geo_partner_view.xml'
  ],
- 'installable': False,
+ 'installable': True,
  'application': True,
  'active': False,
```

Comment:
travis raised this issue:
geoengine_partner/__manifest__.py:5: [C8103(manifest-deprecated-key), ] Deprecated key "active" in manifest file